### ISSUE TITLE
Pytorch Quantizer does not generate .xmodel file?

### ISSUE THREAD

**Issue by NguyenDucTuong** [Xilinx/Vitis-AI#571]
I tried using the Pytorch Quantizer, however, it does not generate the .xmodel file even if I have called export_xmodel(). The code I used is:

from pytorch_nndct.apis import torch_quantizer, dump_xmodel
input = torch.randn([batch_size, 1, 28, 28])
quantizer = torch_quantizer(quant_mode = 'calib', module = model, input_args=(input),
                            state_dict_file=os.getcwd()+'/trained_model.pth')
quant_model = quantizer.quant_model
#quantizer.fast_finetune(evaluate, (quant_model, val_loader, criterion))

quant_set = torch.utils.data.Subset(val_data,list(range(1)))
quant_loader = torch.utils.data.DataLoader(quant_set,batch_size=1,shuffle=False)
for images,_ in quant_loader:
    quant_model(images)

quantizer.export_xmodel(deploy_check=False)

This codes does create a quantize_result folder but there is no .xmodel file in there. 
Any help will be appreciated.

**Comment by niuxjxlnx** [comment]
Hi, @NguyenDucTuong :

It needs to call quantizer.export_quant_config(...) in calibration(quant_mode='calib') stage, then to call quantizer.export_xmodel(...) in test(quant_mode='test') stage.

**Comment by Brady1021** [comment]
It's very helpful,thanks
